examples/shader/post_processing.rs

```diff
@@ -83,7 +87,9 @@ impl Plugin for PostProcessPlugin {
             // you need to extract it manually or with the plugin like above.
             // Add a [`Node`] to the [`RenderGraph`]
             // The Node needs to impl FromWorld
-            .add_render_graph_node::<PostProcessNode>(
+            //
+            // The [`ViewNodeRunner`] is a special [`Node`] that will automatically run the query on the view
```

Comment:
```suggestion
            // The [`ViewNodeRunner`] is a special [`Node`] that will automatically run a query on each view matching the <ViewQuery> trait thing
```
(link to it correctly via rustdoc)